src/infra/core/tests/tests/ingest/test_fetch.rs

```diff
@@ -637,22 +590,10 @@ async fn test_fetch_container_ok() {
     let tempdir = tempfile::tempdir().unwrap();
     let target_path = tempdir.path().join("fetched.bin");
 
-    let content = indoc!(
-        "
-        city,population
-        A,1000
-        B,2000
-        C,3000
-        "
-    );
-
     let fetch_step = FetchStep::Container(FetchStepContainer {
-        image: crate::utils::HttpServer::IMAGE.to_owned(),
-        command: Some(vec!["/bin/bash".to_owned()]),
-        args: Some(vec![
-            "-c".to_owned(),
-            "printf \"city,population\nA,1000\nB,2000\nC,3000\n\"".to_owned(),
-        ]),
+        image: BUSYBOX.to_owned(),
+        command: Some(vec!["printf".to_owned()]),
+        args: Some(vec!["city,population\nA,1000\nB,2000\nC,3000\n".to_owned()]),
```

Comment:
Should we use `CSV_BATCH_OUTPUT` constant here too, now that you separated data from the command?

Reply:
A good catch! Yes, can be replaced -- Done